index.html

```diff
@@ -9,6 +9,7 @@
     <script src="scripts/index.js"></script>
     <link href="https://unpkg.com/aos@2.3.1/dist/aos.css" rel="stylesheet">
     <script src="https://unpkg.com/aos@2.3.1/dist/aos.js"></script>
+    <link href="https://raw.githubusercontent.com/AtharvaK123/Sites-For-All.github.io/main/styles/logo.png" rel="icon" />
```

Comment:
We cannot have a link to your github thing with the image. You need to put the icon file in the styles folder. This is because otherwise the image will not work after some time. Please commit the fix changes and then we can go from there.